.dependabot/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+update_configs:
+  # Keep go modules up to date, batching pull requests weekly
```

Comment:
seems like this should be `js modules` :)

Reply:
doh :)

Reply:
Ooops! My copy and paste is showing!